src/routes/chat/ChatHeader/ChatHeader.jsx

```diff
@@ -87,19 +93,50 @@ const ChatCreateOffer = ({ setOffer }) => {
 
 const ChatHeader = () => {
   const [offer, setOffer] = useState(true);
-
+  const {
+    sellerPrice,
+    buyerPrice,
+    sellerNumberOfShares,
+    buyerNumberOfShares
+  } = useSelector(state => state.chat.chatConversation);
+  const userType = useSelector(state => state.misc.userType);
+  const otherPartyUserType = getOtherPartyUserType({ userType });
+  const userPrice = getUserPrice({
+    userType: otherPartyUserType,
+    sellerPrice,
+    buyerPrice
+  });
+  const userNumberOfShares = getUserNumberOfShares({
+    userType: otherPartyUserType,
+    buyerNumberOfShares,
+    sellerNumberOfShares
+  });
+  const otherPartyPrice = getUserPrice({
+    userType: otherPartyUserType,
+    sellerPrice,
+    buyerPrice
+  });
+  const otherPartyNumberOfShares = getUserNumberOfShares({
+    userType: otherPartyUserType,
+    buyerNumberOfShares,
+    sellerNumberOfShares
+  });
   const createOffer = () => {
     setOffer(true);
   };
   return (
     <div className="chat__header column is-paddingless">
       <div className="columns is-mobile is-marginless">
         <ChatOfferDetails
-          headerText="Seller Offer"
-          quantity="2000"
-          price="6.89"
+          headerText={`${userType === 'seller' ? 'Buyer' : 'Seller'} Offer`}
```

Comment:
We have constants for user types, `BUYER` and `SELLER`. Check `constants/user.js`